### PR TITLE
LPS-70258 There's no policy for displaying organization's logo

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-70258](https://issues.liferay.com/browse/LPS-70258).<br/><br/>/cc @pei-jung @alee8888 @ChrisKian @dejuknow @patriciajeanperez

### CODE REVIEW

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ organizations-types-help=Set this JSON object to change the organizations types.
 use-initials-for-default-user-portrait=Use Initials For Default User Portrait
```
Comment (liferay-appsec) [inline modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
See https://github.com/liferay-appsec/liferay-portal/pull/327

**Pull request closed without merging**